cmd/soroban-rpc/internal/test/docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
     # Note: Please keep the image pinned to an immutable tag matching the Captive Core version.
     #       This avoids implicit updates which break compatibility between
     #       the Core container and captive core.
-    image: ${CORE_IMAGE:-stellar/unsafe-stellar-core:20.2.0-1716.rc3.34d82fc00.focal}
+    image: ${CORE_IMAGE:-stellar/unsafe-stellar-core:20.4.1-1807.b152dc51d.focal}
```

Comment:
should we be using the protocol 21 version of core here?

Reply:
They are only really set through the `CORE_IMAGE` variable